Calibration/LumiAlCaRecoProducers/python/ALCARECOAlCaPCCZeroBiasFromRECO_Output_cff.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+# AlCaReco for track based calibration using MinBias events
+OutALCARECOAlCaPCCZeroBiasFromRECO_noDrop = cms.PSet(
+    SelectEvents = cms.untracked.PSet(
+        SelectEvents = cms.vstring('pathALCARECOAlCaPCCZeroBiasFromRECO')
+    ),
+    outputCommands = cms.untracked.vstring(
+        'keep *_*alcaPCCProducer*_*_*') 
```

Comment:
@Dr15Jones 
is it even happening with the task feature now enabled? Would a dangling loaded module be run triggered by a keep statement if it's not already explicitly listed on a task?